From 258fa821f52f57914168bac76cc2caf6217d4f39 Mon Sep 17 00:00:00 2001
From: Matt Young <matt@mlyoung.cool>
Date: Wed, 22 Oct 2025 14:37:36 +1000
Subject: [PATCH] Fix scip target reference

---
 cmake/check_deps.cmake | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/cmake/check_deps.cmake b/cmake/check_deps.cmake
index b7d5b2c018..d445515260 100644
--- a/cmake/check_deps.cmake
+++ b/cmake/check_deps.cmake
@@ -107,10 +107,10 @@ if(USE_PDLP AND BUILD_PDLP)
 endif()
 
 if(USE_SCIP)
-  if(NOT TARGET SCIP::libscip)
-    message(FATAL_ERROR "Target SCIP::libscip not available.")
+  if(NOT TARGET SCIP::scip)
+    message(FATAL_ERROR "Target SCIP::scip not available.")
   endif()
-  set(SCIP_DEPS SCIP::libscip)
+  set(SCIP_DEPS SCIP::scip)
 endif()
 
 # CXX Test
-- 
2.51.1.dirty

